src/ConsoleMode.ps1

```diff
@@ -0,0 +1,103 @@
+# Hack! https://gist.github.com/lzybkr/f2059cb2ee8d0c13c65ab933b75e998c
+
+if ($IsWindows -eq $false) {
```

Comment:
The api exists on all versions of Windows - but the flag we need is not recognized universally - just Win10 Anniversary and later I think.  The api call will fail (I think harmlessly) if it doesn't recognize the flag.

The workaround should not be needed in V6 - I fixed it in PowerShell: https://github.com/PowerShell/PowerShell/pull/2991